HerPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -13,6 +13,7 @@ public record class Questionnaire
     public string AddressLine2 { get; set; }
     public string AddressLine3 { get; set; }
     public string AddressTown { get; set; }
+    public string AddressCounty { get; set; }

```

Comment:
As mentioned above, Questionnaires are currently stored in the DB so normally you would need to generate a database migration to add this new field/column. Fortunately we can ignore that in this case.